frontend/components/projects/edit/information/config.ts

```diff
@@ -113,8 +114,8 @@ export const projectInformation = {
       validation: {
         required: 'Link url is required',
         pattern: {
-          value: /^https?:\/\/.+\..+/,
-          message: 'Url should start with http(s):// and have at least one dot (.)'
+          value: /^https?:\/\/\S+\.\S+$/,
+          message: 'Url should start with http(s):// and have at least one dot (.) and cannot include white spaces'
```

Comment:
```suggestion
          message: 'Url must start with http(s)://, have at least one dot (.) and cannot include white spaces'
```

Why do we allow http here?